run-compose.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+docker pull pgvector/pgvector:pg16
+
+mkdir -p volume
+
+if [ ! -d "$PWD/volume/postgresql" ]; then
+   mkdir -p volume/postgresql/data
+   chmod 755 -R $PWD/volume/postgresql/
+fi
+
+if [ -z $1 ]; then
+  chmod 755 -R $PWD/volume/
+  docker-compose -f docker-compose.yaml up -d
```

Comment:
afaik docker-compose is deprecated. We should use `docker compose` (that's inside docker)